proxy.go

```diff
@@ -43,25 +38,9 @@ func (p *Proxy) Listen(port int) {
 	if err != nil {
 		log.Fatal(err)
 	}
+
 	defer l.Close()
-	delay := 5 * time.Millisecond
```

Comment:
Thanks @vedhavyas , I think delaying is a good trick if OS is exhausted of file descriptors. Could you leave the delaying part out?

Reply:
ah i misread it